web-next/app/code-review/copilot-code-review/page.module.css

```
@@ -1,1010 +1,1016 @@
+  max-width: 100% !important;
+  padding: 0 0 6rem !important;
+  box-sizing: border-box !important;
 }
 
-@keyframes pulse {
-  0%,
-  100% {
-    opacity: 0.7;
-    transform: scale(1);
+@media (max-width: 960px) {
+  .layout :global(.sidebar) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)